utils/src/main/java/ai/elimu/analytics/utils/AssessmentEventUtil.java

```diff
@@ -0,0 +1,25 @@
+package ai.elimu.analytics.utils;
+
+import android.content.Context;
+import android.content.Intent;
+import android.util.Log;
+
+import ai.elimu.model.enums.analytics.LearningEventType;
+import ai.elimu.model.v2.gson.content.WordGson;
+
+public class AssessmentEventUtil {
+
+    public static void reportWordAssessmentEvent(String packageName, WordGson wordGson, Float masteryScore, Long timeSpentMs, Context context, String analyticsApplicationId) {
```

Comment:
Add documentation with examples, to make it easier for developers to user the method.